swan/scripts/others/userconfig.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Author: Danilo Piparo, Enric Tejedor, Pedro Maximino, Diogo Castro 2023
+# Copyright CERN
+# Here the user session is configured, including the home directory and 
```

Comment:
I think this comment is no longer true. I think it should say something like "Here the software environment for notebooks and terminal is configured, using the LCG release as basis and (possibly) a user environment script".

Reply:
Yeah the comment was outdated and was changed now.